src/compiler/csharp_plugin.cc

```diff
@@ -38,10 +38,14 @@ class CSharpGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
 
     bool generate_client = true;
     bool generate_server = true;
+    bool generate_lite_client = true;
```

Comment:
If we were to have independent options to turn off client and lite client generation I think we should default this to false so there's no bloat for the existing projects.